README.md

```diff
@@ -5,6 +5,11 @@ Implementation of eis_utils in pure Rust.
 
 This crate relies heavily on the functionality already provided by [the iot-identity-services](https://github.com/Azure/iot-identity-service).
 
+# Build
+```
+SOCKET_DIR="/run/aziot" cargo b
```

Comment:
which dependency needs the env var `SOCKET_DIR` set at buildtime?

Reply:
iotedge, iot-identity-service

https://github.com/search?q=repo%3Aomnect%2Fmeta-omnect%20SOCKET_DIR&type=code